app/CocoaPods/CPCLIToolInstallationController.m

```diff
@@ -165,7 +166,8 @@ - (void)setDoNotRequestInstallationAgain;
 
 - (NSURL *)binstubSourceURL;
 {
-  return [[NSBundle mainBundle] URLForResource:@"pod" withExtension:nil];
+  NSURL *bundleURL = [[NSBundle mainBundle] bundleURL];
+  return [bundleURL URLByAppendingPathComponents:@[ @"Contents", @"Helpers", @"pod" ]];
```

Comment:
I don't think you need to write your own `URLByAppendingPathComponents:` extension, probably simpler to just use `fileURLWithPathComponents`?.

```objc
NSString *bundlePath = [[NSBundle mainBundle] bundlePath];
return [NSURL fileURLWithPathComponents:@[bundlePath, @"Contents", @"Helpers", @"pod"]];
```
```